### ISSUE TITLE
You should always be able to submit your submissions, even if the contest has ended. It should just be marked as "suspicious".

### ISSUE THREAD

**Issue by kostka-dev** [sio2project/oioioi#172]
It should work exactly the same as with the submissions send with the submit script. The platform should accept the submission, but not judge it automatically and mark it as "suspicious".

**Comment by A-dead-pixel** [comment]
Could you explain the reasoning behind this?

**Comment by DietPawel** [comment]
Of course,

No matter what contest onsite or online, there is always a big spike of submissions in the last seconds of the contest. It's just the nature of competitive programming.

On every other contest there is a participant claiming he submitted a solution last second, which is not visible. It takes time to see through logs and prove the request was past deadline. If submission is visible to the contestant, marked as suspicious and not judged it would be more transparent to what is wrong, thus more user friendly.

**Comment by twalen** [comment]
most likely we need to add some option (in the contest and/or contest round) to enable/disable this functionality.

**Comment by DietPawel** [comment]
I agree. Even better  would to to customize the grace period from None to infinity.

@NadiaHoffmann I might have some code implementing that in dirty way somewhere If you are interested in that.

**Comment by NadiaHoffmann** [comment]
@DietPawel I could definitely use your piece of code for an inspiration, thank you!

**Comment by kama-pa-go** [comment]
@NadiaHoffmann, you assigned yourself to this issue over two years ago, but there has been no recent activity. Since neither your solution nor the code mentioned by @DietPawel is currently visible in the repository, I have unassigned you to make it clear that the task is open for others. Unfortunately, this means a new contributor will likely need to start from scratch. If you have any work-in-progress code you could share, it would still be very helpful!
